packages/edge-preview-authenticated-proxy/src/index.ts

```diff
@@ -156,9 +156,11 @@ async function handleRawHttp(request: Request, url: URL) {
 				"Access-Control-Allow-Origin": request.headers.get("Origin") ?? "",
 				"Access-Control-Allow-Method": "*",
 				"Access-Control-Allow-Credentials": "true",
-				"Access-Control-Allow-Headers": "x-cf-token,x-cf-remote",
+				"Access-Control-Allow-Headers":
```

Comment:
What about `"Access-Control-Allow-Headers": "*"`?

Reply:
`*` is prone to breaking for our use case due to how it's interpreted with credentialed requests, e.g. if a user tries to send an `Authorization` header. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers#directives